script/docker/linux/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:trusty
+
+RUN apt-get update
+RUN apt-get install --quiet --yes \
```

Comment:
Since GitHub has Cash Dollars, I would try to create a new base image from this, since it's not gonna change very often at all, it'll save you a lot of build time

Reply:
Way ahead of ya: https://hub.docker.com/r/shiftkey/github-desktop/